CRM/Contact/BAO/Group.php

```diff
@@ -337,196 +338,7 @@ public function addSelectWhereClause(?string $entityName = NULL, ?int $userId =
    *   The new group BAO (if created)
    */
   public static function create(&$params) {
-
-    // only check if we need to remove parents if a params was provided
-    $parentsParamProvided = array_key_exists('parents', $params);
-
-    $params += [
-      'group_type' => NULL,
-      'parents' => NULL,
-    ];
-
-    // Fill title and frontend_title if not supplied
-    if (empty($params['id']) && empty($params['title'])) {
-      $params['title'] = $params['frontend_title'] ?? $params['name'];
-    }
-    if (empty($params['id']) && empty($params['frontend_title'])) {
-      $params['frontend_title'] = $params['title'];
-    }
```

Comment:
I see you were able to drop this code due to #31096 😁

Reply:
However, I think some of the test failures are due to missing fallbacks, so I think we need to add more fallbacks to `Group.entityType.php` for `title` and `frontend_title`